### DISCUSSION TITLE
Add automatically installed peer deps to `package.json`

### DISCUSSION THREAD

**Discussion by sachinraja** [npm/feedback#407] [CLI] [answered]
Currently, when I install a package with peer dependencies in npm 7, they will automatically be added to the `package-lock.json`. I think those peer deps should also be added to `package.json`, which would mimic someone manually installing them. It would be nice to see which packages are installed in my project, especially when they are at the top level.

**Comment by MylesBorins** [comment, marked as answer]
@cloudagon if you would like to make a proposal to change how the npm CLI works the next step would be to open an issue over at https://github.com/npm/rfcs

Thanks for the suggestion!
